internal/lifecycle/readme.go

```diff
@@ -65,19 +118,24 @@ func parseREADME(filename, serviceName, gcrURL string) (Lifecycle, error) {
 		return nil, errNoREADMECodeBlocksFound
 	}
 
-	lifecycle, err := codeBlocksTolifecycle(codeBlocks, serviceName, gcrURL)
-	if err != nil {
-		return lifecycle, fmt.Errorf("[lifecycle.parseREADME] transforming code blocks in %s to executable commands: %w", filename, err)
+	var l Lifecycle
+	for _, b := range codeBlocks {
+		c, err := b.toCommands(serviceName, gcrURL)
```

Comment:
Maybe rename `c` to something plural to indicate that it is a slice rather than a single command.

Reply:
Resolved, thank you.